packages/eslint-config/package.json

```diff
@@ -33,6 +33,7 @@
         "npm-run-all": "^4.1.5"
     },
     "devDependencies": {
+        "@comet/eslint-plugin": "workspace:^7.7.0",
```

Comment:
this is already a `"dependency"`

Reply:
absolutely correct. thanks for mentioning it. 

removed it.